lib/LaTeXML/Package/svmult.cls.ltxml

```diff
@@ -85,6 +85,13 @@ DefMath('\gid',    "\x{2267}", role => 'RELOP', meaning => 'greater-than-or-equa
 DefMath('\grole',  "\x{2277}", role => 'RELOP', meaning => 'greater-than-or-less-than');
 Let('\qedsymbol', '\qed');
 
+#  special signs and characters
```

Comment:
The source I used for locating these: https://github.com/openeventdata/Computational-Approaches/blob/master/svmult.cls#L954-L960

The arXiv doc was hitting an error on `\E`